packages/hint-manifest-scoped-urls/src/hint.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * @fileoverview Checks if URLs in the manifest are in scope and accessible
+ */
+
+import { URL } from 'url';
+import { relative } from 'path';
+import { IHint, HintContext, NetworkData, ProblemLocation } from 'hint';
+import { ManifestEvents, ManifestParsed } from '@hint/parser-manifest';
+import { debug as d } from '@hint/utils';
+
+import meta from './meta';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class ManifestScopedUrlsHint implements IHint {
+    public static readonly meta = meta;
+    public constructor(context: HintContext<ManifestEvents>) {
+        /**
+         * See if the `start_url` is accessible.
+         */
+        const startUrlAccessible = async (startUrl: string, resource: string, startUrllocation: ProblemLocation): Promise<boolean> => {
+            let networkData: NetworkData;
+
+            try {
+                networkData = await context.fetchContent(startUrl);
+            } catch (e) {
+                debug(`Failed to fetch ${startUrl}`);
+                const message = `Request failed for 'start_url'`;
+
+                context.report(resource, message, { location: startUrllocation });
+
+                return false;
+            }
+
+            const response = networkData.response;
+
+            if (response.statusCode !== 200) {
+                const message = `Specified 'start_url' is not accessible. (status code: ${response.statusCode}).`;
```

Comment:
We can drop the `()`
```suggestion
                const message = `Specified 'start_url' is not accessible. Status code: '${response.statusCode}'.`;
```